Test/errortests/InvalidArgument1.dfy.expect

```diff
@@ -1,8 +1,62 @@
-Argument '2' not recognized. Must be one of:
-	'3'
-	'4'
-	'migration3to4'
-	'experimentalDefaultGhost'
-	'experimentalDefaultCompiled'
-	'experimentalPredicateAlwaysGhost'
+Description:
```

Comment:
Why is this output changing so much? Seems like something is wrong here.

Reply:
I think it's because the `2>` was replaced with `>`

Reply:
Oops - that change of 2> to > was inadvertent. I changed it back so that the expected output is not dependent on the help text (and contains the error message!).

This does raise a bigger policy question about what dafny output should go to stdout and what to stderr and which information should be checked by the tests. But not a question for this PR.